viper/tests/floats_test_program.rs

```diff
@@ -0,0 +1,98 @@
+use std::fs::File;
```

Comment:
Is this a new file? Why do we need to test Viper from Prusti?

Reply:
Viper tests for floats are very minimal and API is still unstable. Therefore, I think it is a good IDE to have a test that checks the API calls.